cmd/controller/controller.go

```diff
@@ -258,6 +269,19 @@ func isNotFound(err error) bool {
 	return strings.Contains(grpc.ErrorDesc(err), "not found")
 }
 
+func resolveURL(base, ref string) (string, error) {
+	refURL, err := url.Parse(ref)
+	if err != nil {
```

Comment:
I think this version is quite broken.  Please include some tests.

Reply:
This function to resolve a relative chart tarball URL is also in the chart-repo cmd: https://github.com/kubeapps/kubeapps/blob/master/cmd/chart-repo/utils.go#L395

Reply:
Fwiw, I also included a possible solution in my [earlier comment](https://github.com/bitnami-labs/helm-crd/pull/23#discussion_r183299565).  If you don't want to use that version (based around `URL.Parse`) for some reason (older golang?), I suggest you instead use `URL.ResolveReference` to do the heavy lifting.